platform-hub-api/app/services/kubernetes/token_lookup.rb

```diff
@@ -29,18 +31,20 @@ def lookup_static_tokens(token, kind = 'user')
           )
         end
       end
-      nil
+      found.any? ? found : nil
```

Comment:
I thought about that but this would make chaining lookups more fiddly as `[]` is truthy by definition.